BUILD.md

```diff
@@ -0,0 +1,212 @@
+
+![Hyperledger Sawtooth](images/sawtooth_logo_light_blue-small.png)
+
+Hyperledger Sawtooth Core Developers Setup Guide
+=============
+If you are planning to contribute code to please review the contributing guide: [CONTRIBUTING.md]
```

Comment:
This markdown version of the contributing guide is no where near up to date with the contributing guide in the docs.

Reply:
I am unclear how that document is out of date, It links to the documents contributing guide: https://sawtooth.hyperledger.org/docs/core/releases/latest/community/contributing.html
Of course this dissociation of the guide from the code is a little weird since it forces developer to leave the code base for a version of docs that may not be correct for the code they have.

Reply:
My mistake.  I didn't read `CONTRIBUTING.md` closely enough, but judged it based on its length.